app/pb_kits/playbook/pb_avatar/_avatar.jsx

```diff
@@ -4,34 +4,51 @@ import React from 'react'
 import classnames from 'classnames'
 import { map } from 'lodash'
 
+import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
+
 import { Image } from '../'
 
 type AvatarProps = {
+  aria?: object,
   className?: String,
-  name: String,
+  data?: object,
+  id?: String,
   imageUrl: String,
-  size?: 'xs' | 'sm' | 'md' | 'lg' | 'xl',
-  status: 'online' | 'away',
-}
+  name: String,
+  size?: "md" | "lg" | "sm" | "xl" | "xs",
+  status: "away" | "offline" | "online",
+};
 
-const firstTwoInitials = (name) => (
+const firstTwoInitials = (name) =>
   map(name.split(/\s/), (name) => name[0])
     .join('')
     .substring(0, 2)
-)
+    .toLowerCase()
```

Comment:
This would be a breaking, out of context change and not one that I think we would want for this kit. /cc @jasoncypret

Reply:
I changed this code to make it mirror more closely the .downcase implemented in the Ruby method:

https://github.com/powerhome/playbook/blob/d979d99df735547bc88ef36c278c89e3d7d19c5a/app/pb_kits/playbook/pb_avatar/avatar.rb#L17-L19